### ISSUE TITLE
Mac: Not authorized to send Apple events to System Events

### ISSUE THREAD

**Issue by SteveSultan** [ankurk91/google-chat-electron#13]
```
Error: 44:48: execution error: Not authorized to send Apple events to System Events. (-1743)

    at ChildProcess.<anonymous> (/Applications/google-chat-electron.app/Contents/Resources/app.asar/node_modules/applescript/lib/applescript.js:49:13)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
```

---
![error-electron-mac](https://user-images.githubusercontent.com/6111524/117918731-40e8ac80-b309-11eb-8704-526e437d56de.jpg)

google-chat-electron 2.2.0
Electron 12.0.6
darwin 19.6.0
Locale: en-US

**Comment by ankurk91** [comment]
Okay what is your mac os version? 

I will debug it soon. 

meanwhile you can try to grant above permission to app from system preferences.

**Comment by SteveSultan** [comment]
macOS Catalina

Steve Sultan
Full Stack Web Developer
Toronto, ON
phone: 437-997-4553

https://SteveSultan.com
***@***.***
www.linkedin.com/in/stevesultan/<http://www.linkedin.com/in/stevesultan/>
[LinkedIn icon]<https://www.linkedin.com/in/stevesultan/>  [Twitter icon] <https://twitter.com/anonymous_Stv>

________________________________

**Comment by ankurk91** [comment]
@SteveSultan

Unless we find an automated way to fix this issue, you can grant permission manually. See screenshot

![mac-electron-fix-permission](https://user-images.githubusercontent.com/6111524/117918752-4a721480-b309-11eb-9e9e-b4fc6c9a9f01.jpg)

**Comment by ankurk91** [comment]
Fixed in 2.3.0
